vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -338,7 +338,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
         }
     }
 
-    priority: number = 161;
+    priority: number = 171;
```

Comment:
Why you need dependency extension to be after properties extension?

Reply:
Will this break any of your stuff? Basically this plugin grabs `_context` from the properties plugin, which is not defined until after the properties plugin has initialized.